prober/icmp.go

```diff
@@ -234,15 +234,31 @@ func ProbeICMP(ctx context.Context, target string, module config.Module, registr
 	rttStart := time.Now()
 
 	if icmpConn != nil {
+		ttl := module.ICMP.TTL
+		if ttl > 0 {
+			if c4 := icmpConn.IPv4PacketConn(); c4 != nil {
+				level.Debug(logger).Log("msg", "Setting TTL (IPv4 unprivileged)", "ttl", ttl)
+				c4.SetTTL(ttl)
+			}
+			if c6 := icmpConn.IPv6PacketConn(); c6 != nil {
+				level.Debug(logger).Log("msg", "Setting TTL (IPv6 unprivileged)", "ttl", ttl)
+				c6.SetHopLimit(ttl)
+			}
+		}
 		_, err = icmpConn.WriteTo(wb, dst)
 	} else {
+		ttl := 64
```

Comment:
bit uneasy with the magic numner here, can we use value of TTL from `config/config.go:82`.

Reply:
Done in commit 2393796